academy/3-ibc/1-what-is-ibc.md

```diff
@@ -31,7 +31,9 @@ Cross-chain communication between application-specific blockchains in Cosmos cre
 
 IBC is essential for application-specific blockchains like the ones in the Cosmos network. It offers a standard communication channel for applications on two different chains that need to communicate with each other.
 
-Most Cosmos applications execute on their own purpose-built blockchain running their own validator set (at least before the introduction of [Interchain Security](https://blog.cosmos.network/interchain-security-is-coming-to-the-cosmos-hub-f144c45fb035)). These are the application-specific blockchains built with the Cosmos SDK. Applications on one chain may need to communicate with applications on another blockchain, for example an application could accept tokens from another blockchain as a form of payment. Interoperability at this level calls for a method of exchanging data about the state or the transactions on another blockchain.
+<!-- @Cit: include relative link to Interchain Security content -->
```

Comment:
@tmsdkeys to be included once the section on interchain security is reviewed and added, right? If yes, I will make a note of it.

Reply:
Yes, for now I've linked to a blog post from Informal. When #1264 is included, we can link there

Reply:
Perfect! Made a note on it. 😉